core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly

```diff
@@ -34,7 +34,8 @@ THE SOFTWARE.
       <l:task href="${rootURL}/" icon="icon-up icon-md" title="${%Back to Dashboard}"/>
       <l:task href="${rootURL}/manage" icon="icon-gear2 icon-md" permission="${app.ADMINISTER}" title="${%Manage Jenkins}"/>
       <l:task href="new" icon="icon-new-computer icon-md" permission="${createPermission}" title="${%New Node}"/>
-      <l:task href="configure" icon="icon-gear2 icon-md" permission="${app.ADMINISTER}" title="${%Configure}"/>
+      <l:task href="${rootURL}/configureClouds" icon="icon-health-40to59 icon-md" permission="${app.ADMINISTER}" title="${%Configure Clouds}"/>
```

Comment:
is this the only way (except the legacy one) to get to the configure clouds page now? In which case I would not expect Clouds to be managed via "Manage Nodes" and this would be confusing and not pass the path of least surprise.

I think @rsandell had a slightly similar concern.

Reply:
Why would you not configure clouds ("node factories") on "manage nodes"?

Reply:
I don't expect to see tyre factories what I go to get new tyres from the tyre shop...  (sorry I just can't find a better analogy right now).

We have told users that Nodes are the things that run their jobs - not the things that can create things that can run their jobs.  Thus I think this is confusing. (it's not called "Manage Nodes and Node Factories" and thus not obvious)  - I'm not sure what you have against a separate "Manage Clouds" section in the Management area)

Also if I am not mistaken Nodes (Computers) have different Permissions to Clouds (Clouds don't have a separate Permissions from Administer).

Reply:
> I'm not sure what you have against a separate "Manage Clouds" section in the Management area

Seems like a redundant entry when a suitable entry page already exists.

> it's not called "Manage Nodes and Node Factories" and thus not obvious

So, rename this to _Manage Nodes and Clouds_ and done?

Reply:
> So, rename this to Manage Nodes and Clouds and done?

works for me

Reply:
🤔 why do we call it Nodes and not agents anyway?
all our docs is about agents ( I know internally and all)

Reply:
> why do we call it Nodes and not agents anyway?

Because you can also configure master node properties, and master is not an agent.

Remember the term that 'agent' is the replacement for.

Reply:
OT: I really want to kill the Master as a Node....

Reply:
If a Node is something with Executors, then yeah, it'd be great to remove that functionality from the master since it's a poor setup.

Reply:
> If a Node is something with Executors, then yeah, it'd be great to remove that functionality from the master since it's a poor setup.

You can have a node with no executors.  it will just run Flyweight tasks.  However you can not create a Node like that via the UI (except for the master)

Reply:
> yeah, it'd be great to remove that functionality from the master since it's a poor setup.

It's been 0 days since you've experienced how cumbersome it is to set up Jenkins with agents, and now you want to make that mandatory? 😁 

FWIW, if properly analyzed, usage stats would show that by number of instances, 0 agents is the most popular configuration by far. I've seen that a few years ago when I had to manually wrestle the SQLite DB. IOW, we'd break _a lot_ instances.

This is a nice goal, but there's actual work to be done here to even get closer.